api/envoy/extensions/filters/http/grpc_field_extraction/v3/config.proto

```diff
@@ -114,10 +128,10 @@ message FieldExtractions {
   // message Bar {
   //   string name = 1;
   // }
-  map<string, RequestFieldExtraction> request_field_extractions = 1;
+  map<string, RequestFieldDisposition> request_field_extractions = 1;
 }
 
-message RequestFieldExtraction {
+message RequestFieldDisposition {
```

Comment:
This isn't the disposition of the entire field, just the value.  Suggest calling it `FieldValueDisposition`.

Reply:
Changed to `RequestFieldValueDisposition` as we may have `ResponseFieldValueDisposition` which cannot share the same message.